packages/react-composites/src/composites/CallComposite/components/buttons/Custom.tsx

```diff
@@ -48,7 +48,7 @@ export const generateCustomControlBarButtons = (
               styles={buttonProps.styles}
               key={`${buttonProps.placement}_${i}`}
               // set default icon if not provided
-              onRenderIcon={() => <CallCompositeIcon iconName={buttonProps.iconName ?? 'ControlButtonOptions'} />}
+              onRenderIcon={() => <CallCompositeIcon iconName={buttonProps.iconName ?? ''} />}
```

Comment:
Why is this a empty string here now? shouldn't it default to something? or is this so that it doesn't render a icon with a empty prop?